examples/compute-dg-api.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<p>I really like the way you introduce the FEMap in the other notebook, i.e. thinking about edges and nodes. Maybe a long term solution (i.e. not here) might be to merge the two into a single "handling FEMap" notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/cinnabar/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I think this is also content that would best live in a "Concepts" section, along with the high-level explainer of the calculations I mentioned above. Definitely out of scope of this PR though.</p>